geotools-suite/pom.xml

```diff
@@ -134,28 +134,162 @@
 			<plugin>
 				<groupId>org.apache.felix</groupId>
 				<artifactId>maven-bundle-plugin</artifactId>
-				<version>2.3.4</version>
+				<version>3.5.0</version>
 				<extensions>true</extensions>
 				<configuration>
 					<instructions>
 						<Embed-Dependency>
-							*;scope=compile|runtime;artifactId=!jts|gt-opengis|vecmath|jsr-275|geoapi|geoapi-pending|xml-apis|log4j|xercesImpl|xml-apis-xerces|org.osgi.*
+							*;scope=compile|runtime;artifactId=!jts|gt-geojson|gt-opengis|jsr-275|geoapi|geoapi-pending|xml-apis|log4j|xercesImpl|xml-apis-xerces|org.osgi.*
 						</Embed-Dependency>
 						<Embed-Transitive>true</Embed-Transitive>
 						<Bundle-SymbolicName>
 							${project.groupId}.${project.artifactId}
 						</Bundle-SymbolicName>
 						<Bundle-Name>${project.name}</Bundle-Name>
+						<Private-Package>
+							<!-- gt-main -->
+							org.geotools.data;version=${bundle.version};-split-package:=merge-first,
```

Comment:
version=${bundle.version} is equivalent to version >= 19.1 which is not what we want I would think we want to specify a version range.